### PR TITLE
Support --copy mode for clickhouse local as a shortcut for format conversion

### PR DESCRIPTION
**Pull request by denis-hananein**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Support --copy mode for clickhouse local as a shortcut for format conversion https://github.com/ClickHouse/ClickHouse/issues/68503

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

```
clickhouse local --copy < input.json > output.tsv
```

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

**Review by alexey-milovidov — APPROVED** [review]

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 51243d122b865b13fbf96a24628841156686e7da with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/68583/51243d122b865b13fbf96a24628841156686e7da/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/68497/9dee9ecfb4adee4cff099f13afe61bdc2c38170e/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: tests/queries/0_stateless/03228_clickhouse_local_copy_argument.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${CLICKHOUSE_LOCAL} --copy < $CUR_DIR/data_json/twitter.jsonl
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/03228_clickhouse_local_copy_argument.sh:7]: Shellcheck said CUR_DIR does not exist,
there is CURDIR.

Comment (denis-hananein) [reply tests/queries/0_stateless/03228_clickhouse_local_copy_argument.sh]: my bad, fixed

**Comment by denis-hananein** [comment]
1 test failed by stop signal. 
seems unrelated to the pr changes

**Review by alexey-milovidov — APPROVED** [review]

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@denis-hananein, A problem is found: https://github.com/ClickHouse/ClickHouse/issues/70865

**Pull request merged**